astropy/coordinates/matching.py

```diff
@@ -252,6 +264,10 @@ def search_around_sky(coords1, coords2, seplimit, storekdtree='_kdtree_sky'):
     Searches for pairs of points that have an angular separation at least as
     close as a specified angle.
 
+    This is inteded for use on coordinate objects with arrays of coordinates,
+    not scalars.  For scalar coordinates, it is better to use the ``separation``
+    methods.
```

Comment:
hmm, I do like the idea of adding examples of this stuff to the docs.  Let me push up a try at this and see what you think.